server/routerlicious/packages/test-utils/src/test/testsForHistorian.spec.ts

```diff
@@ -29,4 +34,20 @@ describe("Test for TestUtils", () => {
         assert.equal(getCommit.sha, putCommit.sha, "Sha not equal of commits!!");
         assert.equal(getCommit.message, commitParams.message, "Message not equal of commits!!");
     });
+
+    it("Blob Test", async () => {
+        const historian = new TestHistorian();
+        const gitManager = new GitManager(historian);
+        const blob1: ICreateBlobParams = {
+            content: "content",
+            encoding: "utf8",
+        };
+        const blob2: ICreateBlobParams = {
+            content: fromUtf8ToBase64(blob1.content),
+            encoding: "base64",
+        };
+        const createBlobResponse1 = await gitManager.createBlob(blob1.content, blob1.encoding);
+        const createBlobResponse2 = await gitManager.createBlob(blob2.content, blob2.encoding);
+        assert.strictEqual(createBlobResponse1.sha, createBlobResponse2.sha, "Sha for both blobs should match");
```

Comment:
nit: Add a comment saying that since the blob contents are same, only one copy is stored and they both will have the same sha.